valhalla/baldr/landmark.h

```diff
@@ -96,27 +97,13 @@ inline LandmarkType string_to_landmark_type(const std::string& s) {
   return it->second;
 }
 
-inline uint64_t encode_lnglat(double lng, double lat) {
-  return (((uint64_t(lng * 1e7) + uint64_t(180 * 1e7)) & ((1ull << 32) - 1)) << 31) |
-         ((uint64_t(lat * 1e7) + uint64_t(90 * 1e7)) & ((1ull << 31) - 1));
-}
-
-inline std::pair<double, double> decode_lnglat(uint64_t location) {
-  double lng = (int64_t((location >> 31) & ((1ull << 32) - 1)) - 180 * 1e7) * 1e-7;
-  double lat = (int64_t(location & ((1ull << 31) - 1)) - 90 * 1e7) * 1e-7;
-
-  return std::make_pair(lng, lat);
-}
```

Comment:
didnt need to expose these in the header moved them into the body and simplified. here you can see there is clearly something wrong with the decoding step